### ISSUE TITLE
Loss on vggface2 dataset and others

### ISSUE THREAD

**Issue by wfcb-85** [deepinsight/insightface#757]
Hi,
Do you have the loss curve over the training set of the vggface2 dataset ? 
I'm currently training your model on the vggface2 dataset and my error after 5 epochs is around 0.03. However the the performance of that model over the some tests from the same training set is not good. 
Thanks !

**Comment by vitoralbiero** [comment]
Are you using ResNet-100? In my experiments, ResNet-100 does not perform well, seems like it overfits in the VGGFace2 data.
But it works with ResNet-50.

**Comment by williamCorrea85** [comment]
Thanks. I'm using the arcface function on top of anothere pre-trained network I have.
What do you mean by 'it works with ResNet-50' ? What is the training loss that you are obtaining ?
Thanks

**Comment by vitoralbiero** [comment]
I mean that it improves the validation accuracy, but with ResNet-100 it does not.
In the first iteration the loss is 44.8, in middle train (100k iter) is 10.9 and at the end (200k iter) is 1.5.
The model achieves 99.2 on LFW, 91.3 on CFP-FP and 91.9 on AgeDb-30.

But I am training from scratch.
If you are fine-tuning, try changing the learning rate to see if that helps.
